### PR TITLE
Add Changelog for 1.1 and update README

### PR DESCRIPTION
**Pull request by msau42**

### CODE REVIEW

File: CHANGELOG-1.1.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Changelog since v1.0.1
+
+## Breaking Changes
+* Support for the alpha CSINodeInfo CRD has been removed.
+
+## Deprecations
+* Command line flag `--connection-timeout` is deprecated and has no effect.
+* Command line flag `--provisioner` is deprecated and has no effect.
+* Command line flag `--provisioning-retry-count` is deprecated and has no effect.
```
Comment (msau42) [inline CHANGELOG-1.1.md]: @jsafrane these two were actually removed and not using the new deprecatedflags lib. Could you add them back?

Comment (jsafrane) [reply CHANGELOG-1.1.md]: I can see `--provisioner` in current master:
https://github.com/kubernetes-csi/external-provisioner/blob/8a62ab7dd9915f5c56ae5f4664e94e4930034ee0/cmd/csi-provisioner/csi-provisioner.go#L60

And IMO  `--provisioning-retry-count` was never released, I can't find it in any tag.

Comment (msau42) [reply CHANGELOG-1.1.md]: Thanks for checking. I removed the retry count flags

**Pull request merged**